src/service/action-impl.js

```diff
@@ -402,6 +402,17 @@ export class ActionService {
     }
   }
 
+  /**
```

Comment:
@aghassemi does this work? We can possibly refactor lightbox and amp-story to use this function instead.